packages/react/src/components/HelpText/HelpText.tsx

```diff
@@ -54,16 +62,20 @@ export const HelpText = ({
             className={cn(
               classes.helpTextIcon,
               classes.helpTextIconFilled,
+              classes[size],
               className,
             )}
-            title={title}
             data-state={open ? 'open' : 'closed'}
+            aria-hidden={true}
           />
           <Helptext
-            className={cn(classes.helpTextIcon, className)}
-            title={title}
+            className={cn(classes.helpTextIcon, classes[size], className)}
             data-state={open ? 'open' : 'closed'}
+            aria-hidden={true}
           />
+          <div className={utilClasses.visuallyHidden}>
+            {title ? title : 'Toggle help text.'}
+          </div>
```

Comment:
Er det ikke mulig å bruke `title` på `button`-komponenten eller `aria-label` her i stedet?

Reply:
´title´ på button-komponenten legger på en tooltip til knappen når man hoverer over den. Ut ifra noen artikler jeg leste så kan ikke `aria-label` automatisk oversettes av skjermlesere slik som vanlig tekstinnhold i DOMen kan. Jeg har også lest et par steder at man bør foretrekke å bruke native HTML over aria-labels hvor man kan. Jeg har ikke klart å finne noen åpenbare ulemper med visually hidden metoden utenom kanskje vedlikehold av css klassen dersom en eller annen oppdatering skulle påvirke den.

Reply:
"Toggle help text" er for øvrig ikke en spesielt informativ tittel, og alt av tekst burde egentlig komme som input fra utsiden siden designsystemet ikke tar hensyn til språk. Det er bedre å ikke ha noen tittel i det hele tatt hvis den ikke er oppgitt. Hvis dette er ment som en snarvei for å overholde tilgjengelighetskrav, er det bedre å gjøre `title` påkrevd. Vi bør etter hvert få på plass noen retningslinjer på hvorvidt slike ting skal være påkrevd, gjerne etter at vi har tatt noen runder med erfaringsutveksling. Starter vi med å ha den påkrevd, er det i hvert fall bakoverkompatibelt hvis vi skulle løse opp på kravet siden.

Reply:
Det er nok lurt å unngå aria-attributter hvis det kan løses med HTML, ja. Men hvis du beholder den slik, gjelder dette med `div`/`span` også her - `button` er inline og kan derfor bare bestå av andre inline-komponenter :-)